modules/ROOT/pages/anypoint-insights-agentic.adoc

```diff
@@ -0,0 +1,63 @@
+= Monitoring Agentic Assets
+
+//TODO: VERIFY API MANAGER
+In US and EU Cloud instances of Anypoint Platform, navigate from Agent Visualizer or API Manager to view detailed metrics on agentic assets (agent, broker, or MCP server instances).
+
+* Agent: An autonomous software component that uses goals, context, and available tools (often via an LLM) to decide and execute actions on behalf of a user or system.
+* Broker: An intelligent routing service that coordinates task delegation across specialized agents in your enterprise. Brokers originate from MuleSoft and are referenced in the agent network YAML. 
+* MCP server: A service that implements the Model Context Protocol to expose tools and data to AI clients, enabling LLMs to invoke external capabilities through a standard interface.
+
+For information about Agent Visualizer, see xref:agent-visualizer::index.adoc[]. 
+
+//TODO: PREREQS: sub tier, permissions, instances on API Manager or Agent Visualizer? anything else?
+
+//TODO: LOOKBACK settings 24 hours, anything else?
+//TODO: AUTO-REFRESH INTERVALS 
+//TODO: MANUAL REFRESH
+
+//TODO: PROVIDE LINKS TO DOCS WITH INFO ON HOW TO OPEN THE METRICS PAGES
+
+[[properties]]
+== Instance Properties
+
+//TODO: WHERE DO THE LINKS to Runtime Target, Implementation URI, and Proxy URL go?
```

Comment:
@sathishpaulleo  For instance properties (metadata on the pages), can you help with where the links go. Mock doesn't provide info.


```suggestion
//TODO: WHERE DO THE LINKS to Runtime Target, Implementation URI, and Proxy URL go?
```